### PR TITLE
signin and signup pages ui changed

### PR DESCRIPTION
**Pull request by gursharan4312**
### What this PR does (required):
- UI changes for login and signup pages

### Screenshots / Videos (required):
image
image

### CODE REVIEW

**Comment by gursharan4312** [comment]
Demo Login button added to Login Page
image

File: client/src/pages/Login/LoginForm/LoginForm.tsx [hunk 1]
```diff
@@ -29,6 +29,15 @@ interface Props {
 export default function Login({ handleSubmit }: Props): JSX.Element {
   const classes = useStyles();
 
+  const loginDemoUser = async (
+    setFeildValue: (field: string, value: any, shouldValidate?: boolean | undefined) => void,
+    submitForm: (() => Promise<void>) & (() => Promise<any>),
```
Comment (dproc96) [inline client/src/pages/Login/LoginForm/LoginForm.tsx]: Getting an unexpected any warning here, can you try and address that?

**Review by dproc96 — COMMENTED** [review]
Looks good pending the warning I mentioned below

**Comment by gursharan4312** [comment]
Warnings removed

**Review by dproc96 — APPROVED** [review]
LGTM

**Pull request merged**